File: CSharp/1867.cs
using System;
using System.Linq;

class URI {

    static void Main(string[] args) 
    { 
        string[] n;
        char[] algarismo;
        int a;
        int b;
        int soma=0;

        while (true) 
        {
            n = Console.ReadLine().Split();
            if(n[0]=="0" && n[1]=="0")
            {break;}

            


            for(int i=0;i<2;i++)
            {
                while(n[i].Length!=1)
                {
                    algarismo=n[i].ToCharArray();
                    foreach(char c in algarismo)
                    {
                        soma+= int.Parse(c.ToString());
                    }
                    n[i]=soma.ToString();
                    soma=0;
                }
                soma=0;
            }
            a=int.Parse(n[0]);
            b=int.Parse(n[1]);

            if(a>b)
            {Console.WriteLine(1);}
            else if(a<b)
            {Console.WriteLine(2);}
            else{Console.WriteLine(0);}


        }
    }
}

File: Python/1867.py
while True:
    n = input().split()
    if n[0] == "0" and n[1] == "0":
        break

    for i in range(2):
        while len(n[i]) != 1:
            algarismos = list(n[i])
            soma = 0
            for c in algarismos:
                soma += int(c)
            n[i] = str(soma)
    
    a = int(n[0])
    b = int(n[1])

    if a > b:
        print(1)
    elif a < b:
        print(2)
    else:
        print(0)
